### PR TITLE
Create placeholders

### PR DESCRIPTION
**Pull request by eduardconstantin**
- [x] I have read and agree to the [Code of Conduct](https://github.com/danielschuster-muc/potter-db/blob/master/CODE_OF_CONDUCT.md), and have followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).

## Pull Request Type

- [x] Other

## Summary

- Create wizard placeholder to be used in characters page

### CODE REVIEW

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Those images are just awesome! Good job! 

Can you just pull from origin/master, so that you can also implement the image for the spells.
And could you rename the images to **missing_*type***?

**Comment by eduardconstantin** [comment]
Thank you! Sure, I'll make the changes.

File: frontend/src/components/pages/characters/CharacterListItem.js [hunk 1]
```diff
@@ -39,7 +25,7 @@ const CharacterListItem = ({ character }) => {
         <CardMedia>
           <Image
             as="image"
-            src={image || "/images/missing_image.jpg"}
+            src={image || "/images/missing_wizard.svg"}
```
Comment (danielschuster-muc) [inline frontend/src/components/pages/characters/CharacterListItem.js:28]:
```suggestion
            src={image || "/images/missing_character.svg"}
```

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
You should probably also rename the `missing_wizard.svg` file to `missing_character.svg`.

**Comment by eduardconstantin** [comment]
oh, in the script, right

**Review by danielschuster-muc — APPROVED** [review]
Everything‘s good now! Thank you for your contribution and those awesome images!

**Comment by eduardconstantin** [comment]
You're welcome! Glad I could help!

**Pull request merged**